src/integrationTests/communityJest.test.ts

```diff
@@ -0,0 +1,98 @@
+import { Crypto } from '@peculiar/webcrypto';
+import {
+  createCommunity,
+  joinCommunity,
+  getCommunityOwnerData,
+  tryToJoinOfflineRegistrar,
+  assertReceivedCertificates,
+  assertReceivedChannels,
+} from './community';
+import { createApp, createAppWithoutTor } from './utils';
+
+jest.setTimeout(600_000);
+
+const crypto = new Crypto();
+
+global.crypto = crypto;
+
+describe('integration test', () => {
```

Comment:
As Emilia pointed out with Damian's PR, the test label should be more precise. We know that this is integration test so it could be better to say something like e.g. "Testing with Tor"